setup.cfg

```diff
@@ -1,5 +1,2 @@
-[bdist_wheel]
-universal = 1
```

Comment:
technically this is still valid and should be kept -- a universal wheel contains no version-specific code and is pure python (which we still qualify for)

Reply:
It does give a bit of extra protection by naming the file `pyflakes-2.2.0-py3-none-any.whl` instead of `pyflakes-2.2.0-py2.py3-none-any.whl`, and then pip on Python 2 gives this error:

```
ERROR: pyflakes-2.2.0-py3-none-any.whl is not a supported wheel on this platform.
```

And that's before it gets to the `python_requires` check. I think the wheel check has been around longer than the `python_requires` check?

Anyway, I've re-instated this.

---

The `python_requires` check for `pyflakes-2.2.0-py2.py3-none-any.whl` looks like:

```
ERROR: Package 'pyflakes' requires a different Python: 2.7.17 not in '>=3.6'
```

Reply:
Looking at https://packaging.python.org/guides/distributing-packages-using-setuptools/#pure-python-wheels it does appear that we should be removing `universal` because universal is for py2 & py3 otherwise just pure-python wheels

Reply:
@asottile `universal` wheels according to the official packaging docs (see last comment) are meant to advertise support for Py2 and Py3 both. At this point, the package won't actually work on Py2 and advertising support for it is incorrect. Do you have information from the PyPA contradicting their documentation?

Reply:
Comparing wheel filenames created with and without the flag:

```
pyflakes-2.2.0-py2.py3-none-any.whl
pyflakes-2.2.0-py3-none-any.whl
```

Their contents are identical, except for `pyflakes-2.2.0.dist-info/WHEEL` (and `RECORD`'s hash of this file):

```
Wheel-Version: 1.0
Generator: bdist_wheel (0.36.0)
Root-Is-Purelib: true
Tag: py2-none-any
Tag: py3-none-any
```
```
Wheel-Version: 1.0
Generator: bdist_wheel (0.36.0)
Root-Is-Purelib: true
Tag: py3-none-any
```

Reply:
perhaps the nuance of that field has changed over time -- my thought is that for python 4 it would still be correct for pure python packages to claim "universal" status

Reply:
Wait, we're already on Python 4? :stuck_out_tongue:

Reply:
At least right now universal only means 2 + 3.

Shall I remove `universal = 1`?